index.html

```diff
@@ -4,9 +4,58 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Google</title>
+    <link rel="stylesheet" href="main.css">
 </head>
 <body>
+    <nav>
+        <div id="left">
+            <a class="left" href="#">About</a>
+            <a class="left" href="#">Store</a>
+
+        </div>
+        <div id="right">
+            <a class="gmail" href="#">Gmail</a>
+            <a href="#">Images</a>
+            <img id="google-app" src="./googleapp.png" alt="Google apps " title="Google apps">
+            <button type="submit" class="btn">Sign In</button>
+
+        </div>
+    </nav>
+     
+    <div class="google_img">
+        <img src="https://web.archive.org/web/20191130234759im_/https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png"
+         alt="Google">
+
+    </div>
+    <div id="search" >
+        <span>
+            <img class="icon-search" src="./search_black_24dp.svg" alt="">
+        </span>
+        <input type="search"  >
+        <span>
+            <img src="./mic_black_24dp.svg" alt="">
```

Comment:
Please check the indentation and the white space in the code and remove all the white space and make the indentation correct.

Reply:
- [ ] Here you can explain why this is important. Tell something about a correct identation helps to read better the code in the future and solving future issues.

Reply:
Nicely done this time. Thank you for making the requested changes! :clap: :+1: